client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -267,7 +267,7 @@ class Upload extends React.Component {
 				<QueryActiveTheme siteId={ siteId } />
 				{ themeId && complete && <QueryCanonicalTheme siteId={ siteId } themeId={ themeId } /> }
 				<ThanksModal source="upload" />
-				<HeaderCake backHref={ backPath }>{ translate( 'Upload theme' ) }</HeaderCake>
+				<HeaderCake backHref={ backPath }>{ translate( 'Install theme' ) }</HeaderCake>
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Install Themes', 'NULL', { context: 'NULL'} )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/install-action-messaging)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).